hugegraph-llm/src/hugegraph_llm/config/config_data.py

```diff
@@ -70,6 +70,7 @@ class ConfigData:
     graph_user: Optional[str] = "admin"
     graph_pwd: Optional[str] = "xxx"
     graph_space: Optional[str] = None
+    enable_prop_limit: Optional[str] = "False"
```

Comment:
prefer short param  name like `limit_property`

Reply:
changed to `limit_property`